easybuild/easyconfigs/p/PyQtGraph/PyQtGraph-0.13.3-gfbf-2022b.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'PythonPackage'
+name = 'PyQtGraph'
+version = '0.13.3'
+
+homepage = 'http://www.pyqtgraph.org/'
+description = """PyQtGraph is a pure-python graphics and GUI library built on PyQt5/PyQt6/PySide2/PySide6 and numpy."""
+
+toolchain = {'name': 'gfbf', 'version': '2022b'}
+toolchainopts = {'pic': True}
+
+github_account = 'pyqtgraph'
+source_urls = [GITHUB_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['2dd03b68543fba85aea04e3c07311bfe7d5fd87f40ad6954e83b261d23f92de1']
+
+dependencies = [
+    ('Python', '3.10.8'),
+    ('SciPy-bundle', '2023.02'),
+    ('PyOpenGL', '3.1.6'),
+]
+
+use_pip = True
+download_dep_fail = True
+
+sanity_pip_check = True
+# Rather than importing module (without framework) we check for installed directory
+options = {'modulename': False}
```

Comment:
@kosl Can you clarify why this is needed? Is it related to not including `PyQt5` as dependency?

Reply:
Yes. The module can be used with PyQt5 or PySide6.